tfjs-backend-webgpu/src/webgpu_program.ts

```diff
@@ -65,6 +65,10 @@ export const compileProgram =
         layout: 'auto'
       });
 
+      if (printShader) {
+        console.log(program.constructor.name);
```

Comment:
Maybe we can put some decoration around the name so that it can be easily found.

Reply:
Done